extensions/oidc-client/runtime/src/main/java/io/quarkus/oidc/client/runtime/OidcClientImpl.java

```diff
@@ -93,7 +93,8 @@ public OidcClientImpl(WebClient client, String tokenRequestUri, String grantType
                 Uni<HttpResponse<Buffer>> response = request.sendBuffer(OidcCommonUtils.encodeForm(body))
                         .onFailure(ConnectException.class)
                         .retry()
-                        .atMost(3);
+                        .atMost(3)
```

Comment:
Would make sense to make this configurable?

Reply:
@pedroigor Hi Pedro - we already have it indirectly configurable when OIDC connection is established at the start up (`connection-delay` property - it is divided by 2, ex, if it is `20s` then 10 attempts to connect will be made every 2 seconds).
This case is different - there was a test failing (and I've only seen it once) where the connection to KC was dropped - so here it is a try to recover for a very short period of time - this is when a client is awaiting for response, so I was not sure it should be configurable. But I don't really mind - numbers do not look that great :-), I'll have a look